setup.sh

```
@@ -1,97 +1,714 @@
+    fi
+
+    if [ "$build_image" == "build" ]; then
+        download "true"
+    else
+        download "false"
+    fi
+}
+
+function download() {
+    local LOCAL_BUILD=$1
+    cd $SCRIPT_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)